src/test/scala/strawman/collection/immutable/test/ArraySeqTest.scala

```diff
@@ -0,0 +1,865 @@
+package strawman
+package collection
+package immutable
+package test
+
+import org.junit.Assert._
+import org.junit.Test
+
+import scala.{Any, Exception, Int, NoSuchElementException, Nothing, Unit}
+import scala.Predef.{identity, intWrapper}
+import scala.runtime.RichInt
+import scala.util.Try
+import scala.util.control.NonFatal
+
+class ArraySeqTest {
```

Comment:
Could you move this file to the `test/junit/src/test/scala/strawman/collection/immutable/` directory?

Reply:
Sure, I only put it in a subdirectory named "test" because the other tests were also in a "test" subdirectory.